lib/beaker/hypervisor/fusion.rb

```diff
@@ -61,5 +81,48 @@ def cleanup
         @logger.notify "No cleanup for fusion boxes"
       end
 
+      # Set the hostname of all instances to be the hostname defined in the
+      # beaker configuration.
+      #
+      # @return [void]
+      # @api private
+      def set_hostnames
+        @hosts.each do |host|
+          if host['platform'] =~ /el-7/
+            # on el-7 hosts, the hostname command doesn't "stick" randomly
+            host.exec(Command.new("hostnamectl set-hostname #{host.name}"))
+          else
+            host.exec(Command.new("hostname #{host.name}"))
+          end
+        end
+      end
+
+      # Configure /etc/hosts for each node
+      #
+      # @return [void]
+      # @api private
+      def configure_hosts
```

Comment:
I believe that this logic is already contained in host_prebuild_steps::hack_etc_hosts.